app/models/facility.rb

```diff
@@ -109,6 +116,11 @@ def self.parse_facilities(file_contents)
                    pin: row['pin (optional)'],
                    latitude: row['latitude (optional)'],
                    longitude: row['longitude (optional)'],
+                   facility_size: row['size (optional)'],
+                   enable_diabetes_management: row['enable_diabetes_management (true/false)'],
```

Comment:
Will this be the string `"true"/"false"` or the boolean? Do we need to coerce to boolean here? Otherwise, `"false"` is a truthy value. 

As mentioned in my earlier comment, this is a great place to take care of defaulting behavior. Something like:

```ruby
enable_diabetes_management: (row['enable_diabetes_management'] == "true")
# or
enable_diabetes_management: row['enable_diabetes_management'].downcase.in?('t', 'true')
```

Reply:
These boolean fields are type casted appropriately according to the rule set mentioned [here](https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html). So `"false"` would be a `false` as expected. The only defaulting we really need to do is the `nil`, which I've poked deepa about. It's not much to do at all, it's just more stuff happening in the import iykwim.

Reply:
Addressed in 0f9d452. (Slack ref)